.github/workflows/welcome.yml

```
@@ -0,0 +1,10 @@
+name: Post welcome comment
+on:
+  pull_request:
+    types: [opened]
+permissions:
+  pull-requests: write
+jobs:
+  build:
```

actionlint (1.7.7)

"steps" section is missing in job "build"

(syntax-check)